opentech/apply/determinations/views.py

```diff
@@ -335,5 +335,7 @@ def dispatch(self, request, *args, **kwargs):
 
 class DeterminationDetailView(ViewDispatcher):
     admin_view = AdminDeterminationDetailView
-    applicant_view = ApplicantDeterminationDetailView
     reviewer_view = ReviewerDeterminationDetailView
+    partner_view = ApplicantDeterminationDetailView
+    community_view = None
```

Comment:
@frjo Setting it to None will create a bug which will not allow community reviewers to see the determination for submissions submitted by them. What is the requirement for visibility of determination when the user is a community reviewer or a partner?

Reply:
@Parbhat community reviewers as well as partners can reuse the ReviewerDeterminationDetailView.

Everyone that can view a submission should also be able to view its determinations.

Reply:
@frjo Thanks for the information. We have to create new views for partners, community reviewers as these need different checks.